### PR TITLE
Baseball bat recipe

### PR DESCRIPTION
**Pull request by bloodbowel**
#### Summary
Features "Adds a baseball bat recipe"

#### Purpose of change
You can't make a baseball bat in the game. It shouldn't be too hard to do for someone with some basic tools.

#### Describe the solution
1.Add a recipe for a baseball bat. See the image for details.
2.Remove the reference to a plastic handle on baseball bats. You don't need one.

I thought these were treated and was going to add a soaking period but apparently not.

#### Describe alternatives you've considered
Maybe some additional tools should be required?

#### Testing
Tested on my computer in debug mode.

#### Additional context
Screenshot

### CODE REVIEW

File: data/json/recipes/weapon/bashing.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
 [
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "bat",
+    "byproducts": [ [ "splinter", 10 ] ],
```
Comment (Karol1223) [inline data/json/recipes/weapon/bashing.json:6]:
```suggestion
    "byproducts": [ [ "splinter", 30 ] ],
```

You're losing mass

**Review by Karol1223 — CHANGES_REQUESTED** [review]

**Comment by x-qq** [comment]
I think this recipe needs filing 1 (= require sandpaper) and also include flame treatment for durability (= require fire). Real bats also undergo some sort of chemical treatment but I'm not sure how to represent it here.

Reference: https://www.youtube.com/watch?v=zwK4qdbWwdw

**Comment by bloodbowel** [comment]
> I think this recipe needs filing 1 (= require sandpaper), include flame treatment for durability (= require fire) and require wood cutting 2. Real bats also undergo some sort of chemical treatment but I'm not sure how to represent it here.

I've been doing some research to parse out what is the best way to treat the bat. Originally I was going to have it be soaked like a spear shaft, but saw they didn't all do this. Not all are flame tempered either. The oldest method is bone rubbing which is just pressing it down with a hard object. This probably should require better tools including actual files.

**Comment by Maleclypse** [comment]
Yeah go ahead and add either sand paper or wood filing or both as you see fit and this is mergeable.

**Comment by bloodbowel** [comment]
I added the level 1 file. That should be good enough.

**Comment by Brambor** [comment]
Looking at the commit history, this PR should have been `git merge --squash`ed, if the GitHub interface allows it.

**Pull request merged**